torch/testing/_internal/common_utils.py

```diff
@@ -1883,3 +1883,14 @@ def _assertGradAndGradgradChecks(test_case, apply_fn, inputs):
                       torch.double: 1e-5,
                       torch.half: 1e-2,
                       torch.bfloat16: 1e-1}
+
+# Returns True if version is at least required version, False otherwise
+def version_atleast(version, required_version):
```

Comment:
I looked more into this at it looks like PyTorch's pattern is to use LooseVersion for this, see:

https://github.com/pytorch/pytorch/blob/2dff0b3e918530719f7667cb31541f036a25e3f2/tools/setup_helpers/cmake.py#L119

and

https://github.com/pytorch/pytorch/blob/47c65f82231f189f40430b9c9e5b1fde4f7715ef/test/test_spectral_ops.py#L107

That should let us kill this helper and avoid the Stack Overflow citation.

Reply:
Oh. Great! Thanks!